packages/target-electron/src/deltachat/webxdc.ts

```diff
@@ -302,7 +302,8 @@ export default class DCWebxdc {
       })
       setContentProtection(webxdcWindow)
       // reposition the window to last position (or default)
-      webxdcWindow.setBounds(lastBounds || defaultSize, true)
+      const bounds = lastBounds || adjustDefaultSize(defaultSize)
```

Comment:
Compute it _here_.

Reply:
But hmm, maybe we should really adjust the complete bound? E.g.

```js
adjustSize(lastBounds || defaultSize)
```

Reply:
I'd also do that for the lastBounds, they might come from a different screen
```suggestion
      const bounds = adjustDefaultSize(lastBounds || defaultSize)
```

maybe rename the function to remove the default from it's name, maybe some name that represents more the task of "ensure that size does not exceed screen bounds"?

Reply:
yeah I wanted to do this also, but I'm not skilled enough and don't have the energy to wrestle for hours with typescript to make a single function call, it's really getting in my way to write the simplest javascript code :/

Reply:
TypeScript doesn't complain about that change, but then you only return width and height, but you're not going to pass x and y.